Mathlib/Topology/ExtremallyDisconnected.lean

```diff
@@ -314,3 +314,5 @@ instance instExtremallyDisconnected {ι : Type*} {π : ι → Type*} [∀ i, Top
     · rw [sigma_mk_preimage_image' ij]
       exact isOpen_empty
   · continuity
+
+end
```

Comment:
A topmost anonymous section, inside a global `noncomputable section`. I tend to rather remove that vanilla anonymous section instead.

Reply:
Fixed in #14620.